config/swarm-sip.yml

```diff
@@ -22,7 +22,8 @@ os_container_infra_interfaces:
 - "{{ bdn_name }}"
 
 # Swarm SIP gluster source and mount path
-gluster_src: 10.3.0.12:/swarm_vol
+gluster_volume_name: swarm_vol
+gluster_src: 10.3.0.12:/{{ gluster_volume_name }}
```

Comment:
Cute IP but perhaps this could be extracted from the host facts of the gluster mds node?  It ought to be part of the appliance inventory.

Reply:
Sorry this was not supposed to be part of this commit.

Reply:
But point noted for swarm-sip anyhow.